src/main/java/com/exactpro/th2/sailfish/utils/filter/NullFilter.java

```diff
@@ -45,7 +51,10 @@ public FilterOperation getOperation() {
 
     @Override
     public ExpressionResult validate(Object value) throws RuntimeException {
-        return ExpressionResult.create(FilterUtils.isNull(value) == acceptNull);
+        boolean result = FilterUtils.isNull(value) == acceptNull;
```

Comment:
`isNull` method matches the `null` and `NULL_VALUE`. When `checkNullValueAsEmpty` is set to `false` means we should not consider `NULL_VALUE` as an empty one.

Example for `checkNullValueAsEmpty=false`
```
message {
  "null_field": null
}
```
should fail for filter with operation  "null_field" == EMPTY